web/modeshape-web-explorer/src/main/java/org/modeshape/web/BackupExportServlet.java

```diff
@@ -34,18 +35,19 @@
 public class BackupExportServlet extends HttpServlet {
 
     private static final long serialVersionUID = 1L;
-
+    private File tempDir;
+    
     @Override
     protected void doGet(HttpServletRequest request, HttpServletResponse response)
             throws IOException {
-        File file = new File("zzz");
+        File file = new File(tempDir.getAbsoluteFile() + File.separator + "zzz");
```

Comment:
Probably would be good to add something (e.g., something time-related) to the temporary file to help eliminate chances to clash if 2+ separate users each instigated a backup at overlapping times.